script.js

```diff
@@ -1,4 +1,77 @@
+const totalPlayers = 2
+const diceRollPair = []
+
+let rollDice = function() {
+  randomNum = (Math.random() * 6) + 1
+  randomDice = Math.floor(randomNum)
+  return randomDice
+}
+
+
+for (let i=0; i<totalPlayers; i++) {
+  diceRollPair.push([rollDice(), rollDice()])
+  // jsonObj['Player' + (i + 1)] = diceRollPair[i]
+  // console.log(jsonObj)
+}
+
+let playerIndex = 0
+const playersNumbers = []
+
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
+  let myOutputValue = getPlayerChoice(input)
   return myOutputValue;
 };
+
+
+const getPlayerChoice = function(input) {
+    console.log('getPlayerChoice', input)
+  
+    if (playerIndex < totalPlayers) {
+      const rollPair = diceRollPair[playerIndex]
+      const playerNumber = playerIndex + 1
+      if (!input) {
+        return `Player ${playerNumber} rolled ${rollPair[0]} for Dice 1 and  ${rollPair[1]} for Dice 2. <br>`
+      } else {
+        let tensIndex
+        let onesIndex
+        if (input === '1') {
+          tensIndex = 0 // first dice is tens
+          onesIndex = 1 // second dice is ones
+        } else {
+          tensIndex = 1 // second dice is tens
+          onesIndex = 0 // first dice is ones
+        }
+        makeTens = rollPair[tensIndex] * 10
+        makeNum = makeTens + rollPair[onesIndex]
```

Comment:
like the use of arrays here. Though it might need more message explaination for the players